searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -281,16 +280,28 @@ Matcher::match(const SearchRequest &request,
         reply = std::move(result->_reply);
         SearchReply::Coverage & coverage = reply->coverage;
         if (wasLimited) {
+            LOG(debug, "was limited, degraded from match phase");
             coverage.degradeMatchPhase();
         }
         if (my_stats.softDoomed()) {
+            LOG(debug, "soft doomed, degraded from timeout");
             coverage.degradeTimeout();
         }
-        coverage.setActive(metaStore.getNumActiveLids());
+        uint32_t numUsedLids = metaStore.getNumUsedLids();
+        uint32_t numActiveLids = metaStore.getNumActiveLids();
+        size_t spaceEstimate = mtf->match_limiter().getDocIdSpaceEstimate();
+        LOG(debug, "num used lids = %d", numUsedLids);
+        LOG(debug, "num active lids = %d", numActiveLids);
+        LOG(debug, "space Estimate = %zd", spaceEstimate);
+        size_t estimate = std::min(static_cast<size_t>(numActiveLids), spaceEstimate);
```

Comment:
the space estimate will count both inactive and unused lids, so this comparison makes little sense.